ios/health_kit_reporter.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
   s.platform              = :ios, '9.0'
 
   s.dependency 'Flutter'
-  s.dependency 'HealthKitReporter'
+  s.dependency 'HealthKitReporter', '3.0.0'
```

Comment:
Its probably a good idea to have it pinned to a version, otherwise it would automatically update for users (even if it had breaking changes)